src/MultiColor/MultiColor.test.js

```diff
@@ -0,0 +1,60 @@
+import MultiColor from "./MultiColor";
+import React from "react";
+import { render } from "@testing-library/react";
+
+/**
+ * Test wrapper for MultiColor (provides state for the value)
+ */
+const MultiColorWithState = ({
+  onChange,
+  rows: rowsIn = [
+    {
+      color: "rgba(0,22,252,1)",
+      value: 2
+    }
+  ],
+  ...rest
+}) => {
+  const [rows, setRows] = React.useState(rowsIn);
+  const handleChange = newRows => {
+    setRows(newRows);
+    onChange && onChange(newRows);
+  };
+  return (
+    <MultiColor {...rest} autoHeight onChange={handleChange} rows={rows} />
+  );
+};
+
+function getColumnValues(colIndex) {
+  return Array.from(
+    document.querySelectorAll(`[role="cell"][data-colindex="${colIndex}"]`)
+  ).map(node => node.textContent);
+}
+
+/**
+ * Tests
+ */
+describe("MultiColor", () => {
+  test("can select an item", () => {
```

Comment:
Are you sure this is a test for 'can select an item'? I don't see any selection going on. Looks like you are just rendering MultiColor with some data and checking the value column is as expected. Good test, bad name. You might also want to check the color column appears as expected too.